src/abstractsynchronizer.js

```diff
@@ -89,6 +89,10 @@ olcs.AbstractSynchronizer.prototype.synchronize = function() {
  */
 olcs.AbstractSynchronizer.flattenLayers =
     function(layer, foundLayers, foundGroups, opt_zIndices) {
+  foundLayers.push(layer);
```

Comment:
Closure should have rejected pushing `ol.layer.Base` to an array of `ol.layer.Layer`.